Makefile

```diff
@@ -714,6 +714,11 @@ test-cli-test-case-selector-flag: $(CLI_BIN)
 test-cli-registry: $(CLI_BIN)
 	@$(CLI_BIN) test ./test/cli/registry --registry
 
+.PHONY: test-cli-scenario-to-cli
+test-cli-scenario-to-cli: $(CLI_BIN)
+	@$(CLI_BIN) test ./test/cli/scenario_to_cli --registry
```

Comment:
should be `scenarios_to_cli` from the path I'm seeing, right?

Reply:
Yeah I also think the same but when did **scenarios_to_cli**  then the cli tests failed as you can see in my commit [de92640](https://github.com/kyverno/kyverno/pull/8055/commits/de92640b40edc591f7f439ec1c0c1443c8c695d9).  So should I change it to scenarios_to_cli or not?